### DISCUSSION TITLE
✅ Does WebSSH has support for triggering wake-on-lan?

### DISCUSSION THREAD

**Discussion by jamesaustin** [isontheline/pro.webssh.net#382] [Q&A] [answered]
I would like to be able to `systemctl suspend`  on my remote machine and then use WebSSH to wake-on-lan before SSHing.

Is this something WebSSH supports?

**Comment by isontheline** [comment]
Hello @jamesaustin 👋

I have opened an issue several months ago : https://github.com/isontheline/pro.webssh.net/issues/154

Could you explain your workflow? Would you like a WOL tool inside WebSSH or do you want to send a WOL packet before to connect to a SSH connection?

**Comment by mbierman** [comment]
I would love to be able to send a WOL if the device doesn’t respond to the ssh and then have it automatically retry to ssh. Maybe have a limit on how long it tries before telling the user that something isn’t working as expected. 

Hope this makes sense.

**Reply by isontheline** [reply]
It make sense! Thank you a lot for your feedback @mbierman 🙏

**Comment by bpeterme** [comment]
I would like to second the request for a feature to send WOL packets to my home network. The idea is that I want to ssh into my server on demand, but I don’t wish lo leave the server on all day. I'm currently using a separate app for WOL, but I think it would be thematically very close and convenient to have such a feature in WebSSH.

**Comment by isontheline** [comment]
Hello @bpeterme 👋

Which app are you currently using for WOL?

**Reply by bpeterme** [reply]
It‘s an iOS app called „Wake Me Up“.

**Comment by isontheline** [comment, marked as answer]
Now available on the App Store
